### ISSUE TITLE
How to know  the testset and trainset.

### ISSUE THREAD

**Issue by ling9981** [roboflow/ODinW-RF100-challenge-issues#2]
I searched the official competition website, but I am still confused about the test set that submitted the results and training set. I downloaded the RF100 dataset, but I don't know how to split the training and testing sets for the competition.
